oxd-server/src/main/java/org/gluu/oxd/server/RestResource.java

```diff
@@ -28,197 +31,270 @@ public RestResource() {
     @Path("/health-check")
     @Produces(MediaType.APPLICATION_JSON)
     public String healthCheck() {
-        return "{\"status\":\"running\"}";
+        try (Scope orderSpanScope = TracingUtil.buildSpan("health-check", true)) {
+            TracingUtil.setTag("end-point", "/health-check");
+            return "{\"status\":\"running\"}";
+        }
     }
 
     @POST
     @Path("/get-client-token")
     @Produces(MediaType.APPLICATION_JSON)
     @Consumes(MediaType.APPLICATION_JSON)
     public String getClientToken(String params) {
-        return process(CommandType.GET_CLIENT_TOKEN, params, GetClientTokenParams.class, null);
+        try (Scope orderSpanScope = TracingUtil.buildSpan(CommandType.GET_CLIENT_TOKEN.toString(), true)) {
```

Comment:
we can put this tracing wrapper inside `process`. We have CommandType, so we know everything we need.
In case you wish to log path, pass `@Context UriInfo uriInfo` and then `uriInfo.getAbsolutePath()`.